### PR TITLE
changing server.rb to more closely resemble mhv sandbox

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

**Comment by ayaleloehr** [comment]
@saneshark PTAL. DRYed up my old copy-pasta, added our endpoints back in so it can still be used by our front-end developers, and took out the SSL setup as it seemed to run with SSL on Heroku already with the old configuration (and I didn't have SSL_{KEY,CERT}_PATH set up locally anyways...

File: server.rb [hunk 1]
```diff
@@ -1,57 +1,74 @@
 require 'webrick'
+require 'webrick/https'
 require 'json'
 
 # default port to 3000 or overwrite with PORT variable by running
 # $ PORT=3001 ruby server.rb
-port = ENV['PORT'] ? ENV['PORT'].to_i : 3004
+port = ENV['PORT'] ? ENV['PORT'].to_i : 443
```
Comment (saneshark) [inline server.rb]: may or may not want to change this back to 3004 as well. If anyone else wants to pull this and run it locally 443 will require running as sudo and you'd need to be passing env variables in your sudoers files.

File: server.rb [hunk 2]
```diff
@@ -1,57 +1,74 @@
...
 
-puts "Server started: http://localhost:#{port}/"
+cert_name = [
```
Comment (saneshark) [inline server.rb]: this is also probably no longer necessary

**Comment by saneshark** [comment]
@ayaleloehr other than comments, LGTM

File: server.rb [hunk 3]
```diff
@@ -1,57 +1,74 @@
 require 'webrick'
+require 'webrick/https'
```
Comment (saneshark) [inline server.rb]: and this

**Comment by ayaleloehr** [comment]
Great catches. Addressed and merging. For the record, I do think if we're running this locally then having the SSL code in there makes sense and I'm glad there is a commit that captures it so we can re-add it later if deemed necessary!

**Pull request merged**